source/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve --host 0.0.0.0",
+    "start": "ng serve --proxy-config proxy.conf.json",
```

Comment:
Would you please explain why did you remove the original content of this line? It was an asset to containerization.